test/test_common/utility.h

```diff
@@ -400,6 +400,16 @@ class ScopedFdCloser {
   int fd_;
 };
 
+// TODO(sbelair2) Perform the fd close the close of the IoHandle
```

Comment:
Sorry rephrase? Why is this needed exactly?

Reply:
Yes, that's not clear at all. Here's what I changed it to:

// TODO(sbelair2) Perform the fd close in the close of the IoHandle-
// i.e., ScopedIoHandleCloser should incorporate the ScopedFdCloser